test/index.js

```diff
@@ -551,7 +551,7 @@ function camelize(text) {
  */
 
 function main(argv) {
-  var opt = parseArg();
+  var opt = parseArg(argv);
```

Comment:
You could do the check here. If `argv` is undefined `opt` should be set to `{}`

Reply:
I just added it to get the test error to go away and didn't remove it after I added your solution.